packages/ui5-info/src/ui5-info.ts

```diff
@@ -0,0 +1,432 @@
+import cmp from 'semver-compare';
+import { coerce, major, minor, valid } from 'semver';
+import type { UI5VersionFilterOptions, UI5VersionOverview, UI5VersionsResponse, UI5Version } from './types';
+import { UI5Info, FioriElementsVersion } from './types';
+import { CommandRunner } from './commandRunner';
+import axios from 'axios';
+import { FE_MIN_UI5_VERSION_V2, FE_MIN_UI5_VERSION_V4 } from '@sap-ux/fiori-elements-writer';
+import type { Logger } from '@sap-ux/logger';
+import { ToolsLogger } from '@sap-ux/logger';
+import ui5VersionsFallback from './ui5VersionFallback';
+
+/**
+ * Lowest UI5 version to return (not necessarily the min supported version)
+ */
+export const DEFAULT_MIN_UI5_VERSION = '1.65.0';
+const LatestVersionString = 'Latest';
+const DefaultVersion = LatestVersionString;
+
+export const DEFAULT_UI5_VERSIONS = [
+    DefaultVersion,
+    '1.104.0',
```

Comment:
This should be extended with newer versions and remove no longer supported versions
https://sapui5.hana.ondemand.com/versionoverview.html